mscore/musescore.cpp

```diff
@@ -3659,7 +3662,11 @@ static bool convert(const QString& inFile, const QJsonArray& outFiles, const QSt
 
 static bool convert(const QString& inFile, const QString& outFile)
       {
-      return convert(inFile, QJsonArray{ outFile });
+      if(pluginMode) {
+            return convert(inFile, QJsonArray{ outFile}, pluginName);
+      }else{
+            return convert(inFile, QJsonArray{ outFile });
+      }
```

Comment:
Wrong indentation, please read the [coding principles](https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/musescore-coding-rules).

Reply:
better just drop the braces